tests/test_board.py

```diff
@@ -21,21 +21,37 @@ def test_can_occupy_a_space(board, space):
     assert board.get_spaces()[space] == 'x'
 
 
-def test_knows_when_no_spaces_are_occupied(board):
+def test_returns_false_when_no_spaces_are_occupied(board):
     assert board.is_full() is False
 
 
-def test_knows_when_board_is_not_full(board):
+def test_returns_false_when_board_is_not_full(board):
     num_of_spaces_minus_one = len(board.get_spaces()) - 1
     for _ in range(num_of_spaces_minus_one):
         board.place_marker(_, 'x')
     assert board.is_full() is False
 
 
-def test_knows_when_all_spaces_are_occupied(board):
+def test_returns_true_when_all_spaces_are_occupied(board):
     for _ in range(len(board.get_spaces())):
```

Comment:
Although `_` can be used to represent an unused variable, we're using it to pass each position in this test.

```
>>> _ = 1
>>> 1 + _
2
```

I accept that the value of the variable is mostly irrelevant to the test and we're just creating a full board. However, I find this current implementation is a tad confusing at first glance.

Reply:
How might you re-write this so that the code reads `for position in board:`?